### PR TITLE
LL-6863 - Rebranding - Flexbox component - Add *-gap proprieties

### PR DESCRIPTION
**Pull request by jgiamporcaro-ledger**
Add *-gap proprieties to the Flexbox component.

### Type

Improvement

### Context

[LL-6863]

### Parts of the app affected / Test plan

[LL-6863]: https://ledgerhq.atlassian.net/browse/LL-6863

### CODE REVIEW

File: ui-lib/src/components/Layout/Flex/index.ts [hunk 1]
```diff
@@ -1,8 +1,10 @@
-import { flexbox } from "styled-system";
+import { flexbox, compose } from "styled-system";
 import styled from "styled-components";
+import gapsSystem from "@ui/styles/system/gaps";
 
 const FlexBox = styled.div`
   display: flex;
-  ${flexbox};
+  ${compose(gapsSystem, flexbox)}
```
Comment (jelbaz-ledger) [inline ui-lib/src/components/Layout/Flex/index.ts:7]: :trollface: you removed a semicolon at the end of the line

Comment (jelbaz-ledger) [reply ui-lib/src/components/Layout/Flex/index.ts:7]: (which is ok since it's the last css property in the block)

**Review by jelbaz-ledger — DISMISSED** [review]
LGTM

**Review by juan-cortes — DISMISSED** [review]

**Comment by gre** [comment]
conflict @jgiamporcaro-ledger

**Pull request merged**